src/tcpstream.c

```diff
@@ -203,6 +202,7 @@ void *run_ntttcp_sender_tcp_stream( void *ptr )
 		}
 
 		/* get the local TCP ephemeral port number assigned to this socket, for logging purpose */
```

Comment:
Can you please update the comment too to reflect the purpose. here, we:
"get the local TCP port number assigned to this socket, for logging purpose"

Reply:
Done